packages/components/src/components/pagination/_pagination.scss

```diff
@@ -42,6 +42,19 @@ $css--helpers: true;
     @include carbon--breakpoint('md') {
       overflow: initial;
     }
+
+    // mobile friendly pagination
+    @include carbon--breakpoint-down('md') {
+      .#{$prefix}--pagination__left > *,
+      .#{$prefix}--pagination__right > * {
+        display: none;
+      }
+
+      .#{$prefix}--pagination__items-count,
+      .#{$prefix}--pagination__control-buttons {
+        display: initial;
```

Comment:
Wondering if this perhaps needs to be `display: flex`? Caught a small issue on very small breakpoints in Percy. 

<img width="939" alt="Screen Shot 2020-07-29 at 9 45 25 AM" src="https://user-images.githubusercontent.com/11928039/88828705-765c4900-d180-11ea-976c-3fdc1313e68e.png">


```suggestion
        display: flex;
```

Reply:
Interesting, I think this is actually incorrect in Percy. At least this is what I see at 320px.

<img width="394" alt="Screenshot 2020-07-29 at 20 03 20" src="https://user-images.githubusercontent.com/6061509/88836148-93bc0200-d1d6-11ea-9cbe-a7c5b15a56cd.png">